jquery.littleGallery.js

```diff
@@ -71,30 +74,20 @@
 
             gallery = $('.littleGallery');
 
-            var headHtml = '<span class="littleGallery-img" data="' + data.front.url + '">' + data.front.name + '</span>';
-
-            if (data.back) {
-                headHtml += '<span class="littleGallery-img" data="' + data.back.url + '">' + data.back.name + '</span>';
-            }
-
-            if (data.uv_front) {
-                headHtml += '<span class="littleGallery-img" data="' + data.uv_front.url + '">' + data.uv_front.name + '</span>';
-            }
-
-            if (data.uv_back) {
-                headHtml += '<span class="littleGallery-img" data="' + data.uv_back.url + '">' + data.uv_back.name + '</span>';
-            }
-
-            if (data.variable_uv_front) {
-                headHtml += '<span class="littleGallery-img" data="' + data.variable_uv_front.url + '">' + data.variable_uv_front.name + '</span>';
-            }
-
-            if (data.variable_uv_back) {
-                headHtml += '<span class="littleGallery-img" data="' + data.variable_uv_back.url + '">' + data.variable_uv_back.name + '</span>';
+            if (!$.isEmptyObject(data)) {
+                var headHtml = '';
+                for (var key in data) {
+                    if (data.hasOwnProperty(key)) {
```

Comment:
may be use jQuery.each() (http://api.jquery.com/jquery.each/) and check url and name property existance?